question/bank/viewcreator/templates/creator_display.mustache

```diff
@@ -0,0 +1,34 @@
+
+{{!
+    This file is part of Moodle - http://moodle.org/
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template qbank_viewcreator/creator_display
+    Example context (json):
+    {
+        "displaydata": [
+          {
+                "haslabel": false,
+                "nameclasses": "class-name class-name",
+                "creator": "Creator name",
+                "date": "Date created",
```

Comment:
please use a real data in the date, example doesnt match

Reply:
Real data used for examples and removed useless logic